### PR TITLE
Sample pull request

### PR DESCRIPTION
**Pull request by giorgiosironi**
Only open to allow Atomist experiments to take place.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Anyone here? (posted through Slack)

File: README.md [hunk 1]
```diff
@@ -60,4 +60,6 @@ Testing
 
 You can run the full automated test suite from the base folder with:
 
-`$ ./project_tests.sh`
+```
+$ ./project_tests.sh
+```
```
Comment (giorgiosironi) [inline README.md:65]: Comment on Github

Comment (giorgiosironi) [reply README.md:65]: Another comment

Comment (giorgiosironi) [reply README.md:65]: Another comment now that we can link the repository

**Review by seanwiseman — APPROVED** [review]
🤖

**Comment by giorgiosironi** [comment]
Can be closed now :wrench:

**Pull request closed without merging**